modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Engine.scala

```diff
@@ -24,11 +24,11 @@ object Engine {
       Engine(q.sequences.mapValues(_.map(f)))
   }
 
-  case class State(conditions: Conditions, sequences: Map[Sequence.Id, Sequence.State])
+  case class State(conditions: Conditions, operator: Option[Operator], sequences: Map[Sequence.Id, Sequence.State])
```

Comment:
I'm fine with an extra parameter here, next parameter we add more we could group them all in a single parameter like `metadata` or something like that. @cquiroz I think you already suggested that, am I right?